nifi-extension-bundles/nifi-cdc/nifi-cdc-mysql-bundle/nifi-cdc-mysql-processors/src/test/java/org/apache/nifi/cdc/mysql/processors/CaptureChangeMySQLTest.java

```diff
@@ -500,6 +501,165 @@ public void testExtendedTransaction() {
         testRunner.getProvenanceEvents().forEach(event -> assertEquals(ProvenanceEventType.RECEIVE, event.getEventType()));
     }
 
+    @Test
+    public void testInsertUpdateDeleteSignedAndUnsigned(@Mock Connection connection) throws JsonProcessingException {
```

Comment:
It would be helpful to break this method into some smaller private methods for better maintainability.